app/logic/searchStudents.py

```diff
@@ -0,0 +1,32 @@
+from app.models.user import User
+from app.controllers.main import main_bp
+from flask import json, jsonify
+
+
+
+def searchVolunteers(query):

```

Comment:
Is there a test for this function? I thought there was, but I'm not seeing it in this PR for some reason